Makefile

```diff
@@ -82,6 +82,7 @@ export GIT_PAGER :=
 TAR_XFORM_FLAG = $(shell $(TAR) --version | grep -q GNU && echo "--xform=s" || echo "-s")
 
 GIT_DIR := $(shell git rev-parse --git-dir 2> /dev/null)
+LDFLAGS += -s -w
```

Comment:
I was able to run `COCKROACH_CPUPROF_INTERVAL=1m ./cockroach start` which produced cpuprofile files in the logs directory with contents that looked correct:

```
$ go tool pprof cpuprof.2017-03-17T14_53_33.095
Entering interactive mode (type "help" for commands)
(pprof) top10
570ms of 960ms total (59.38%)
Showing top 10 nodes out of 295 (cum >= 20ms)
      flat  flat%   sum%        cum   cum%
     160ms 16.67% 16.67%      160ms 16.67%  crypto/sha512.block
     110ms 11.46% 28.12%      210ms 21.88%  [cockroach]
      90ms  9.38% 37.50%       90ms  9.38%  [libpthread-2.24.so]
      50ms  5.21% 42.71%       50ms  5.21%  runtime.futex
      40ms  4.17% 46.88%       40ms  4.17%  runtime.scanobject
      30ms  3.12% 50.00%       30ms  3.12%  runtime._ExternalCode
      30ms  3.12% 53.12%       50ms  5.21%  runtime.mallocgc
      20ms  2.08% 55.21%       20ms  2.08%  crypto/elliptic.p256Sqr
      20ms  2.08% 57.29%       20ms  2.08%  runtime.entersyscall
      20ms  2.08% 59.38%       20ms  2.08%  runtime.mapaccess2_fast64
```

So I think it's good as is. The point of this patch (for me) is faster builds during my normal work. If it just sped up the release build time I wouldn't care too much.